src/gui/userstatusselectormodel.cpp

```diff
@@ -406,23 +406,23 @@ QString UserStatusSelectorModel::timeDifferenceToString(int differenceSecs) cons
     } else if (differenceSecs < 60 * 60) {
         const auto minutesLeft = std::ceil(differenceSecs / 60.0);
         if (minutesLeft == 1) {
-            return tr("1 minute");
+            return tr("%1 minute", "", .arg(minutesLeft));
         } else {
-            return tr("%1 minutes").arg(minutesLeft);
+            return tr("%1 minutes", "", .arg(minutesLeft));
         }
     } else if (differenceSecs < 60 * 60 * 24) {
         const auto hoursLeft = std::ceil(differenceSecs / 60.0 / 60.0);
         if (hoursLeft == 1) {
-            return tr("1 hour");
+            return tr("%1 hour", "", .arg(hoursLeft));
```

Comment:
@Valdnet @mgallien This, and other occurrences of a pattern `tr("%1 whatever", "", .arg(n))` would not compile. Having a `.arg(n)` inside the `tr()` is not a valid C++ costruction.

One way to fix this is to use the third argument in the `tr()` without `.arg`. E.G. `tr("%1 minute", "", minutesLeft).arg(minutesLeft)`, instead of `tr("%1 minute", "", .arg(minutesLeft))`. So, the third argument is just `minutesLeft`, then, the `.arg()` call is done with `minutesLeft` to replace `%1`.

Another way, which is documented here https://doc.qt.io/archives/qq/qq19-plurals.html is to use `%n` instead of `%1` (only if there is only one argument). E.G.  `tr("%n minute", "", minutesLeft)`.

Reply:
this is not compiling as shown by the automated checks triggered on the PR
that line should be written like this
```
return tr("%1 hour", "", hoursLeft).arg(hoursLeft);
```

please fix all other calls